src/builtins/BuiltinTemporal.cpp

```diff
@@ -209,13 +307,29 @@ void GlobalObject::installTemporal(ExecutionState& state)
     m_temporalPlainDate = new NativeFunctionObject(state, NativeFunctionInfo(strings->lazyDate(), builtinTemporalPlainDateConstructor, 4), NativeFunctionObject::__ForBuiltinConstructor__);
     m_temporalPlainDate->setGlobalIntrinsicObject(state);
 
-    m_temporalPlainDatePrototype = new TemporalCalendar(state, m_objectPrototype);
+    m_temporalPlainDatePrototype = new Object(state, m_objectPrototype);
```

Comment:
Please change it to `PrototypeObject` like below
(`m_objectPrototype` would be set as proto by default)
```js
m_temporalPlainDatePrototype = new PrototypeObject(state);
```